src/OpenTK.Mathematics/MathHelper.cs

```diff
@@ -67,6 +67,69 @@ public static class MathHelper
         /// </summary>
         public const float Log2E = 1.442695041f;
 
+        /// <summary>
+        /// Returns the sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Sine of the angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double Sin(double a) => Math.Sin(a);
+
+        /// <summary>
+        /// Returns the hyperbolic sine of the specified angle.
+        /// </summary>
+        /// <param name="a">An angle, measured in radians.</param>
+        /// <returns>Hyperbolic sine of the specified angle. If a is equal to NaN, NegativeInfinity, or PositiveInfinity, this method returns NaN.</returns>
+        public static double HSin(double a) => Math.Sinh(a);
```

Comment:
I think that one should be Sinh, as I never so hsin being used in the math world(correct me If I'm wrong?)

Reply:
+1

Reply:
Fixed in most recent comit